examples/html/index.html

```diff
@@ -21,6 +21,15 @@
     <h1>UMA Auth Client html demo</h1>
     <p>Click the UMA Connect button to get started.</p>
     <script src="../../packages/uma-auth-client/dist/uma-auth-client-web-components.umd.cjs"></script>
-    <uma-connect-button></uma-connect-button>
+    <uma-connect-button
+      nostr-pubkey="npub1scmpzl2ehnrtnhu289d9rfrwprau9z6ka0pmuhz6czj2ae5rpuhs2l4j9d"
```

Comment:
Maybe good to call this app-identity-pubkey to be specific? Wouldn't want clients to think it's a user npub or something.

Reply:
updated to app-identity-pubkey